Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,39 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '@{{plainid}}\.'
+      scope: storage.modifier.annotation.scala
+      push:
+        - match: '{{plainid}}\.'
+          scope: storage.modifier.annotation.scala
+        - match: '{{plainid}}'
+          scope: storage.modifier.annotation.scala
+          set: annotation-parameters
+    - match: '@{{plainid}}'
+      scope: storage.modifier.annotation.scala
+      push: annotation-parameters
+
+  annotation-parameters:
```

Comment:
The same `meta.` scope should be added here as in the match in the `annotation` context.